src/Kiosk.Abstractions/Models/Events/EventResponse.cs

```diff
@@ -0,0 +1,16 @@
+using Kiosk.Abstractions.Enums;
+
+namespace Kiosk.Abstractions.Models.Events;
+
+public class EventResponse
```

Comment:
you probably could change name of this dto to something like `GetEventResponse` or `FindEventResponse`